robottelo/hosts.py

```diff
@@ -1259,6 +1260,19 @@ def register_to_cdn(self):
                 f'Error during pool attachment, command output: {cmd_result.stdout}'
             )
 
+    def ping_host(self, host):
+        """Check the provisioned host status by pinging the ip of host
+
+        :param host: IP address or hostname of the provisioned host
+        :returns: None
+        :raises: : `ProvisioningCheckError` if the host is not pingable
+        """
+        result = self.execute(
+            f'for i in {{1..60}}; do ping -c1 {host} && exit 0; sleep 20; done; exit 1'
```

Comment:
This seems like a weird way to do this. Why not do this instead?

`f'ping -c 60 -i 20 {host}'`

Reply:
Calling ping like that won't stop once the first echo is returned.